### PR TITLE
[bump] client: 2.0.0-internal.2.0.1 => 2.0.0-internal.2.0.2 (patch)

### PR DESCRIPTION
**Pull request by jatgarg**
Bumped client from 2.0.0-internal.2.0.1 to 2.0.0-internal.2.0.2.
Also updated the lock file with newer package versions.

### CODE REVIEW

File: lerna-package-lock.json [hunk 1]
```diff
@@ -2138,25 +2138,25 @@
 			}
 		},
 		"@fluidframework/aqueduct": {
-			"version": "2.0.0-internal.2.0.0",
-			"resolved": "https://registry.npmjs.org/@fluidframework/aqueduct/-/aqueduct-2.0.0-internal.2.0.0.tgz",
-			"integrity": "sha512-MBdHVYcsQZ2kNj779TvR9bje8H9BjE30YNoYU1txaTSKAoOUlPSQmFokN6S/Zrguyiyaigv1N7GzxDoyjS9Y9g==",
+			"version": "2.0.0-internal.2.0.1",
```
Comment (CraigMacomber) [inline lerna-package-lock.json:2141]: All these lock file changes seem like a separate thing than the version bump to 2.0.2 (these are updating to the 2.0.1 published packages).

If you want to keep these in the same PR, the description should probably include this.

Comment (CraigMacomber) [reply lerna-package-lock.json:2141]: Note that this explains why this version bump hits 201 files unlike the last one which it 200

Comment (jatgarg) [reply lerna-package-lock.json:2141]: The description and changes are generated automatically by tyler tool. I would like to keep the changes but can modify the description.

Comment (jatgarg) [reply lerna-package-lock.json:2141]: Updated the description

**Review by CraigMacomber — APPROVED** [review]

**Pull request merged**